### ISSUE TITLE
Amplify CLI push/pull no longer working

### ISSUE THREAD

**Issue by Bigbrotha12** [aws-amplify/amplify-cli#11347]
### Before opening, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-adminui/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-adminui/blob/main/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### App Id

dib2j3imz104v

### Region

ap-northeast-2

### Environment name

staging

### Amplify CLI Version

10.4.0

### If applicable, what version of Node.js are you using?

v16.17.0

### What operating system are you using?

Mac

### Browser type?

Brave

### Describe the bug

Attempting to run "amplify push" to push changes to my cloud Backend from within VS Code amplify CLI. 
CLI outputs below message and opens browser to login to Amplify Studio.
```
⠸ Confirm login in the browser or manually paste in your CLI login key:
```
I use my AWS account and login but I'm immediately redirected to error page saying:

There were issues with logging in. If this is a new app, please try again after a few minutes.

This is not a new app and everything was working correctly 2 days.

### Expected behavior

Running "amplify push" should deploy my local changes to backend cloud environment.

### Reproduction steps

1. Open workspace containing Amplify functions on VS Code.
2. Run "amplify push".
3. After being redirected to Amplify Studio Log In page for logging in, log in with AWS account.
4. After logging in, I'm redirected to Amplify Studio Log In page once again which then changes to error page.

### Project Identifier

dab67ccf779b810f660fd6450d8ab872

**Comment by danielleadams** [comment]
Hi @Bigbrotha12 do you only have this issue when you are working in VSCode? Does this happen in another terminal?

**Comment by Bigbrotha12** [comment]
Hi @danielleadams, so far I only used VSCode so haven't tried in other terminals. I tried uninstalling globally and re-installing, also downgrading to v10.0.0 so far hasn't worked for me. Will try again now to re-install v10.0 and use a different terminal see if anything changes.

**Comment by Bigbrotha12** [comment]
@danielleadams, so I tried using a different terminal and encountered the same issue. I also tested creating a brand new project via "amplify init" and it worked both in system terminal and VSCode. I was able to create api and functions on this new project and push them to cloud. I can see the new project when I log into the amplify studio console. It is only when I try to pull or push to my existing project that I run into this authentication issue.
